src/taco-cli/cli/run.ts

```diff
@@ -88,6 +91,12 @@ class Run extends commands.TacoCommandBase {
             if (!remoteConfig) {
                 throw errorHelper.get(TacoErrorCodes.CommandRemotePlatformNotKnown, platform);
             }
+            
+            // DeviceSync/LiveReload not compatible with remote
+            var deviceSync = commandData.options["livereload"] || commandData.options["devicesync"];
+            if (deviceSync) {
```

Comment:
But what if the intention of the user is to run live reload on remote target too ? Will this prevent live-reload from running on remote targets ?